scripts/e2e/ios-devices.js

```diff
@@ -0,0 +1,37 @@
+const DEFEAULT_APPLE_MODEL = 'iPhone 17';
+const DEVICE_MODEL = process.env.E2E_APPLE_DEVICE || DEFEAULT_APPLE_MODEL
```

Comment:
`DEVICE_MODEL` is not really a constant here. I'd prefer it being extracted to a function and computed there.